metadata/src/main/java/edu/unc/lib/dl/event/PremisLogger.java

```diff
@@ -0,0 +1,90 @@
+package edu.unc.lib.dl.event;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Date;
+import java.util.UUID;
+
+import org.apache.jena.riot.RDFDataMgr;
+import org.apache.jena.riot.RDFFormat;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.hp.hpl.jena.rdf.model.Model;
+import com.hp.hpl.jena.rdf.model.ModelFactory;
+import com.hp.hpl.jena.rdf.model.Resource;
+import com.hp.hpl.jena.util.FileManager;
+
+import edu.unc.lib.dl.fedora.PID;
+import edu.unc.lib.dl.rdf.Premis;
+import edu.unc.lib.dl.util.PremisEventBuilder;
+
+public class PremisLogger {
+	private static final Logger log = LoggerFactory
+			.getLogger(PremisLogger.class);
+	public File file;
+	public PID pid;
+	public PremisEventBuilder premisEventBuilder;
+	public Model model;
+	public String eventId;
+	public String cdrEventURI = "http://cdr.lib.unc.edu/event/";
+	
+	public PremisLogger(PID pid, File file) {
+		this.pid = pid;
+		this.file = file;
+		this.eventId = generateUUID();
+	}
+	
+	public PremisEventBuilder buildEvent(Resource eventType, Date date) {
```

Comment:
Since date is being passed in as null every time so far, I think we should add an extra version of this method
public PremisEventBuilder buildEvent(Resource eventType)

Use that everywhere, and directly mention in the javadoc that the event will be assigned the current time as a timestamp